lumeer-core/src/main/java/io/lumeer/core/task/FunctionTask.java

```diff
@@ -90,8 +89,13 @@ public LinkType getLinkType() {
       return linkInstances;
    }
 
-   public List<FunctionTask> getParents() {
-      return parents;
+   @Override
```

Comment:
I would move the parent to the Task interface and these methods to the AbstractContextualTask.